src/views/List/List.jsx

```diff
@@ -51,46 +51,37 @@ export default function List({ token }) {
 
   const onChange = async (listItem) => {
     //when an item is checked, we map through the data array and update the React state
-    const {
-      isActive,
-      id,
-      frequency,
-      lastPurchasedAt = Date.now(),
-      timesPurchased,
-      daysSinceLastPurchase,
-    } = listItem;
-    console.log('listItem:', listItem);
+    const { isActive, id, frequency, lastPurchasedAt, timesPurchased } =
```

Comment:
🔍 This looks like a big change, but it's just a bunch of imports reformatted onto one line. Notably, we no longer assign a default value to `lastPurchasedAt`.

Reply:
Curious: is there a 'preferred' convention when destructuring? (i.e., one line v. each on separate lines)

Reply:
This change was made by Prettier, the auto-formatting tool. There’s a stylistic preference to use multiple lines after a certain line-length. Because i made the list of properties shorter, Prettier moved them back to one line.